arrow-annotations-processor/src/main/java/arrow/optics/OpticsProcessor.kt

```diff
@@ -55,12 +58,20 @@ class OptikalProcessor : AbstractProcessor() {
                 .getElementsAnnotatedWith(optionalsAnnotationClass)
                 .map(this::evalAnnotatedElement)
 
+        annotatedBounded += roundEnv
+                .getElementsAnnotatedWith(boundAnnotationClass)
+                .map(this::evalAnnotatedElement)
+
         if (roundEnv.processingOver()) {
             val generatedDir = File(this.generatedDir!!, "").also { it.mkdirs() }
             LensesFileGenerator(annotatedLenses, generatedDir).generate()
             PrismsFileGenerator(annotatedPrisms, generatedDir).generate()
             IsosFileGenerator(annotatedIsos, generatedDir).generate()
             OptionalFileGenerator(annotatedOptional, generatedDir).generate()
+
+            LensesFileGenerator(annotatedBounded, generatedDir).generate()
+            OptionalFileGenerator(annotatedBounded, generatedDir).generate()
```

Comment:
Why is `LensesFileGenerator` and `OptionalFileGenerator` added twice?

Reply:
Bounded requires that Lenses and Optionals are also generated - this is just in case when only class is only annotated with [at]bounded